tests/components/steam_online/test_config_flow.py

```diff
@@ -150,11 +150,10 @@ async def test_options_flow(hass: HomeAssistant) -> None:
             result["flow_id"],
             user_input={CONF_ACCOUNTS: [ACCOUNT_1, ACCOUNT_2]},
         )
-    await hass.async_block_till_done()
+        await hass.async_block_till_done()
 
     assert result["type"] == data_entry_flow.FlowResultType.CREATE_ENTRY
     assert result["data"] == CONF_OPTIONS_2
-    assert len(er.async_get(hass).entities) == 2
```

Comment:
I took this out because it the mocking wasn't sufficient here now that the platform is setup in time so the test wasn't actually testing the expected thing